src/app/platform/layout.tsx

```diff
@@ -26,6 +27,7 @@ const Layout = ({ children }: { children: React.ReactNode }) => {
   return (
     <div className="min-h-screen flex flex-col bg-blue-50">
       <Navbar />
+      <Sidebar />
```

Comment:
I think Sidebar should be used in `keys/page.tsx` because the side bar items are specific to pages.